fl4health/clients/basic_client.py

```diff
@@ -67,9 +73,13 @@ def process_config(self, config: Config) -> Tuple[Union[int, None], Union[int, N
         if ("local_epochs" in config) and ("local_steps" in config):
             raise ValueError("Config cannot contain both local_epochs and local_steps. Please specify only one.")
         elif "local_epochs" in config:
+            # Initialize total_epochs attribute with 0 if None
+            self.total_epochs = 0 if self.total_epochs is None else self.total_epochs
```

Comment:
I didn't really handle this in my original implementation, but it occurred to me that with this setup, we're assuming that the config never mixes epochs and steps. That is, we could do 2 epochs for the first 10 server rounds and then 100 steps for the next 10. I think the rest of the code functions perfectly fine with that, but we'll have some issues here. 

Maybe we should simply keep track of total steps and log that regardless of whether we're doing epochs or steps?

Reply:
Updated in [31d94bf](https://github.com/VectorInstitute/FL4Health/pull/61/commits/31d94bf2464f27115b4672a5fae1a25a9c95e7fb)!